Drawsana/DrawsanaView.swift

```diff
@@ -140,6 +140,9 @@ public class DrawsanaView: UIView {
     ]
     selectionIndicatorView.layer.actions = [
       "transform": NSNull(),
+      "lineWidth": NSNull(),
+      "lineDashPattern": NSNull(),
+      "cornerRadius": NSNull(),
```

Comment:
What's the reasoning for disabling these animations?

Reply:
If you don't disable them, then every single layer property update triggers an animation, which in this case is wrong because we're moving it directly with the user's finge.